tests/page/page-keyboard.spec.ts

```diff
@@ -482,6 +482,17 @@ it('should support simple copy-pasting', async ({ page, isMac, browserName }) =>
   expect(await page.evaluate(() => document.querySelector('div').textContent)).toBe('123123');
 });
 
+it('should support simple cut-pasting', async ({ page, isMac, browserName }) => {
```

Comment:
why is browserName called here? it's not used

Reply:
Good catch. Updated.

Reply:
cc @dgozman not sure why re-requesting review removed you from the list. Don't think I have permission to add you so ping manually.